docs/modules/ROOT/pages/runbooks/mariadbgalera/MySQLGaleraClusterDown.adoc

```diff
@@ -0,0 +1,24 @@
+= Alert rule: MySQLGaleraClusterDown
+
+== icon:glasses[] Overview
+
+Checks if a Galera Cluster is down.
+This could be caused by full PVC, OOM event, network issues or underlying infrastructure issues.
+
+== icon:bug[] Steps for Debugging
+
+See xref:how-tos/mariadbgalera/debug.adoc[How to debug MariaDB Galera] on how to determine the cluster status and specifically xref:how-tos/mariadbgalera/bootstrap_cluster.html[Boostrap a cluster] on how to bring the cluster back up.
+
+[WARNING]
+====
+Make sure you boostrap from the *last* healthy pod. There are multiple options to find the last healthy pod:
+
+- Create a debug mariadb to mount the PVC and follow these https://mariadb.com/kb/en/galera-cluster-recovery/[instructions].
+
+====
+
+IMPORTANT: If the cluster is down, due to full PVC storage, you first have to resize the underlying PVC.
+This needs to be done by the customer, by choosing a higher instance tier.
```

Comment:
External dependencies (stuff we can't fix ourselves) should never escalate on-call.

Reply:
Well it's almost impossible to automatically figure out based on metrics, what the exact issue is when an instance goes down.

Reply:
I agree, getting woken up by a non-actionable alert is not coll.

Reply:
I removed that section from the docs again and adjusted our alert a bit to not trigger in this case.